m4_sessions/1808-inning/stress_management.md

```diff
@@ -23,10 +30,24 @@
      Share resources in your slack channel
      All sessions NEED to have a closing activity, reflection, discussion
     
-## Introductons 
+## Introductions
+* who we are
 ## Intro to new material
+* What is stress and how we manage it.
+
 ## Small Group Activity
+after introductions, we will break up into three even groups, each of which will be lead by one of us.
+* Guided Meditation *Quin
+  *Everyone in the group will be asked to lie down on a yoga mat and close their eyes. Relaxing music will be playing and I will read a guided meditation.
+* Gardening *Theresa
+  *Students will work as individual to decorate and/or plant a pot.  Music playing also. Pots can be kept at school or taken home.  At end each student will share likes and how to incorporate into their future.
```

Comment:
Similar to Jimmy's section, is there some research that you can highlight on the benefits of creating things/making things?